core/src/com/ericc/the/game/KeyboardController.java

```diff
@@ -37,11 +48,64 @@ public boolean keyDown(int keycode) {
                 break;
             case Input.Keys.SPACE:
                 player.currentAction.action = Actions.NOTHING;
+                break;
             default:
                 break;
         }
 
         logicEngine.update(1);
         return false;
     }
+
+    /**
+     * Actions that player takes and should not affect the turn counter.
+     * @return true if the action that should be taken should not update the turn counter
+     */
+    private boolean playersGUIActions(int keycode) {
+        switch (keycode) {
+            case Input.Keys.MINUS:
+                zoomCamera(this.cameraChange);
+                break;
+            case Input.Keys.PLUS:
+                zoomCamera((-1) * this.cameraChange);
+                break;
+            default:
+                return false;
+        }
+
+        return true;
+    }
+
+    /**
+     * Zooms the camera by the given value (either out or in)
+     * @param zoom the value to add to a camera zoom
+     */
+    private void zoomCamera(float zoom) {
+        if (canZoom(camera.zoom + zoom)) {
```

Comment:
to IMHO w ogóle nie jest odpowiedzialność tej klasy żeby zarządzać zoomem, wywalmy to do osobnej klasy